### PR TITLE
OEP-8: algorithm updates

### PR DESCRIPTION
**Pull request by kremalicious**

### CODE REVIEW

File: 8/v0.4/README.md [hunk 1]
```diff
@@ -276,15 +276,23 @@ Depending on the asset type (dataset, algorithm), there are different metadata a
...
-| **`format`**        | no        | Packaging format of the software. |
-| **`version`**       | no        | Version of the software. |
-| **`container`**     | yes       | Object containing Docker container image, tag to use and the entrypoint for the image|
-| **`files`**         | no        | Array of files implementing custom changes on the image. |
+| Attribute           |   Type                |   Required  | Description                                         |
+| ------------------- | ----------------------| ----------- |--------------------------------------------------- |
+| **`language`**      | `string`              | no          | Language used to implement the software |
+| **`format`**        | `string`              | no          | Packaging format of the software. |
+| **`version`**       | `string`              | no          | Version of the software. |
+| **`files`**         | Array of files object | no         | Array of `File` objects including the encrypted file urls. Further metadata about each file is stored, see [File Attributes](#file-attributes)
```
Comment (kremalicious) [inline 8/v0.4/README.md]: how is this supposed to be used? What is the difference to the files under `main.files`? Do we need this?

File: 8/v0.4/ddo-example-algorithm.json [hunk 2]
```diff
@@ -80,26 +80,18 @@
             "language": "scala",
             "format": "docker-image",
             "version": "0.1",
+            "entrypoint": "ocean-entrypoint.sh",
```
Comment (kremalicious) [inline 8/v0.4/ddo-example-algorithm.json]: the table doesn't mentioned these 2 anymore. Are they used or not?

File: 8/v0.4/README.md (hunk 1 above)
Comment (alexcos20) [reply 8/v0.4/README.md]: sorry, you are right.  files section is already in main

File: 8/v0.4/ddo-example-algorithm.json (hunk 2 above)
Comment (alexcos20) [reply 8/v0.4/ddo-example-algorithm.json]: removed

**Pull request merged**